### ISSUE TITLE
Disable Autoplay

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/atlas#2390]
- [ ] Disable Autoplay on NFT details page.
- [ ] Disable autoplay when user is watching comments

Currently it always autoplays.

**Comment by toiletgranny** [comment]
I went through all designs for the Video page to make sure we're not missing any circumstance under which the autoplay should be disabled, and I also checked how is this problem solved today by our dear colleagues over at YouTube, and _other_ video sharing platforms. 

As a result, I added a new piece of documentation [to our Video player user stories in Figma.](https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=3473%3A253022) For your convenience, I'm pasting it below as well.

> Under circumstances listed below, the countdown for the redirect should be paused and reset, with the Autoplay button component (1) being rendered in the Paused state automatically.
Once the below circumstances are no longer true, the countdown should not be resumed.
The list of circumstances:
> 1. The main Video player component is _completely_ outside of the user's viewport (eg. the user is reading comments underneath the video).
> 2. There's a modal overlay displayed (eg. the NFT workspace allowing the user to list an NFT, the NFT purchase view allowing the user to purchase or bid for an NFT, the Change price dialog, the Accept bid dialog, etc.).
> 3. There's a blockchain transaction processing in the background (eg. for posting a comment, a reaction, or a reaction to a comment).

@dmtrjsg @kdembler Let me know if you think this is vague or if you believe this should be tackled somehow else (from the design perspective).

Personally, I'm uncertain of the feasibility of the second circumstance listed above, from a technical perspective. Is there anything we could use today to just set up one rule/exception for pausing the autoplay countdown whenever there's any kind of modal overlay displayed? Or do we have to set up such rules/exceptions for each modal separately? The latter has some obvious shortcomings and might result in us revisiting this problem every time we add something new to the Video page, so even if we don't have a straightforward technical solution to achieve this goal today, perhaps it's worth investing some time in this.
